data/sql/updates/pending_db_world/rev_1643395587559675400.sql

```diff
@@ -0,0 +1,10 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1643395587559675400');
+
+DROP TABLE IF EXISTS `instance_saved_go_state_data`;
+CREATE TABLE IF NOT EXISTS `instance_saved_go_state_data` (
+  `id` int unsigned NOT NULL COMMENT 'id = instance.id',
+  `entry` int unsigned NOT NULL COMMENT 'entry = gameobject.entry',
+  `guid` int unsigned NOT NULL COMMENT 'guid = gameobject.guid',
+  `state` int unsigned DEFAULT '0' COMMENT 'state = gameobject.state',
```

Comment:
```suggestion
  `id` INT UNSIGNED NOT NULL COMMENT 'instance.id',
  `entry` INT UNSIGNED NOT NULL COMMENT 'gameobject_template.entry',
  `guid` INT UNSIGNED NOT NULL COMMENT 'gameobject.guid',
  `state` INT UNSIGNED DEFAULT '0' COMMENT 'gameobject.state',
```

Reply:
Actually, when I am thinking about it. Why do we store entry? Does it really serve any purpose?